config/Betterloadingscreen/tips/en_US.txt

```diff
@@ -2,11 +2,10 @@
 #Thanks to 0lafe, boubou19, chochem, interestingusernam3, glowredman, GTNH-Afx237v7, leagris, LewisSaber, Nvidiot, Regian24, richardhendricks, yukieiji, FijiAura and others for tip suggestions
 #
 #Maximal characters for a tip: 77 (more can go out of the screen)
-#The line below, including the "#" are 77 chars
-#Adventurer's backpack? Did you mean integrated crafting grid, bed and fluid
+#Use the line below as a reference
+############################################################################
 #
 Infinite water sources are disabled
-Don't give ideas to 0lafe
```

Comment:
May be controversial. I think these player references (other than maybe to Dream or Boubou) get stale and cause confusion for new players

Reply:
0lafe is not longer active and i guess he will not return so delet this tip is fine